components/analytics-solutions-common/org.wso2.analytics.solutions.common/widgets/DateTimeRangePicker/src/GranularityModeSelector.jsx

```diff
@@ -17,8 +19,8 @@
  */
 
 import React from 'react';
-import {IconButton, FlatButton, Popover, Menu} from 'material-ui';
-import {HardwareKeyboardArrowRight, HardwareKeyboardArrowLeft} from 'material-ui/svg-icons';
+import { IconButton, FlatButton, Popover, Menu } from 'material-ui';
```

Comment:
use named imports

Reply:
This code is no longer in user after PR https://github.com/wso2/analytics-solutions/pull/270